css/style.css

```diff
@@ -378,18 +384,651 @@ footer {
   font-family: Poppins, sans-serif;
   font-style: normal;
   font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
+  font-size: 5vw;
   color: #92dcdc;
 }
 
 .get-resume img {
-  padding-left: 16px;
-  height: 18px;
-  width: 18px;
+  padding-left: 5vw;
+  height: 5.6vw;
+  width: 5.6vw;
   align-self: center;
 }
 
 .social img {
   align-items: center;
 }
+
+/* Desktop version */
+
+@media (min-width: 1000px) {
+  body {
+    background-image: none;
+  }
+
+  .flex-nav a {
+    display: none;
+  }
+
+  .menu {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 1.111vw;
+    color: #1a2236;
+    height: 7vw;
+    align-items: center;
+    border: 1px solid #dddfe6;
+  }
+
+  .menu li {
+    margin: 0 3.47vw;
+  }
+
+  .menu .line-hello {
+    background-color: #1a2236;
+    height: 0.14vw;
+    width: 2vw;
+  }
+
+  .menu .line-portfolio {
+    display: none;
+    background-color: #1a2236;
+    height: 0.14vw;
+    width: 2vw;
+  }
+
+  .menu .line-about {
+    display: none;
+    background-color: #1a2236;
+    height: 0.14vw;
+    width: 2vw;
+  }
+
+  .menu .line-contact {
+    display: none;
+    background-color: #1a2236;
+    height: 0.14vw;
+    width: 2vw;
+  }
+
+  main {
+    flex-direction: row;
+    height: 61vw;
+  }
+
+  .geometry img {
+    display: none;
+  }
+
+  .geometry {
+    background-image: url('../assets/img/image-geometry_3.png');
+    background-position: top right;
+    background-repeat: no-repeat;
+    background-size: 48.75vw 61vw;
+    flex: 1 1 50vw;
+  }
+
+  .main-text {
```

Comment:
- [ ] Great work applying the background image for this section. But, there's missing one. Please refer to the comment above about following the given design and look at the images attached there. Also, kindly refer to the design template and make the corresponding changes.

Reply:
fixed 👍